content/chat/rooms/history.textile

```diff
@@ -63,7 +63,7 @@ if let next = try await paginatedResult.next {
 ```
 
 ```[kotlin]
-val paginatedResult = room.messages.get(orderBy = NewestFirst)
+var historicalMessages = room.messages.get(orderBy = OrderBy.NewestFirst)
 println(historicalMessages.items.toString())
 
 while (historicalMessages.hasNext()) {
```

Comment:
Perhaps worth a code comment here to say that because historical messages is paginated, we can iterate through, just to make it clear?